main.py

```diff
@@ -505,6 +509,24 @@ def has_capability(self, capability):
     def get_capability(self, capability):
         return self.capabilities.get(capability)
 
+    def shutdown(self):
+        try:
+            self.send_request(Request.shutdown(), self.exit)
+            self.process.wait(timeout=5000)
+        except OSError as err:
+            exception_log("Error shutdown server", err)
```

Comment:
:+1: We MUST show errors when language servers misbehave. Please change the message to "Error shutting down server".